src/scripts/main.js

```diff
@@ -20,6 +20,29 @@ const food = {
 
 const tree = document.querySelector('#tree');
 
+function createTree(element, data) {
+  const list = document.createElement('ul');
+
+  Object.keys(data).forEach((key) => {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+    list.appendChild(li);
+
+    if (Object.keys(data[key]).length > 0) {
+      createTree(li, data[key]);
+    }
+  });
+
+  element.appendChild(list);
+
+  return list;
+}
+
+createTree(tree, food);
+
+tree = document.querySelector('#tree');
```

Comment:
The `tree` variable is declared as a constant on line 21. Reassigning it here will cause an error. Consider removing this line or changing the variable declaration to `let` if reassignment is necessary.